content/en/docs/instrumentation/js/resources.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Resources"
+weight: 6
+description:
+---
+
+A [resource][] is an immutable representation of the entity producing telemetry
+as attributes. For example, a process producing telemetry that is running in a
+container on Kubernetes has a Pod name, it is in a namespace and possibly is
+part of a deployment which also has a name. All three of these attributes can be
+included in the resource.
+
+In your observability backend you can use the information stored in the resource
+to see to which container (or pod, or deployment, etc.) your traces belong and
+if they have issues (like out of memory, network outages, etc.) you can quickly
+understand why your traces might be slower or errounous.
```

Comment:
```suggestion
In your observability backend, you can use resource information to better investigate
interesting behavior. For example, if your trace or metrics data indicate latency in your
system, you can narrow it down to a specific container, pod, or kubernetes deployment.
```
I kind of prefer framing this up in terms of investigating behavior, but I'm pretty neutral on this suggestion.